--- libevent-2.0.21-stable/configure.in.am13~	2012-11-18 07:34:13.000000000 +0100
+++ libevent-2.0.21-stable/configure.in	2013-01-14 00:35:43.388407165 +0100
@@ -7,12 +7,13 @@ dnl
 dnl Original version Dug Song <dugsong@monkey.org>
 
 AC_PREREQ(2.59c)
-AC_INIT(event.c)
+AC_INIT([libevent], [2.0.21-stable])
+AC_CONFIG_SRCDIR([event.c])
 
 AC_CONFIG_MACRO_DIR([m4])
 
-AM_INIT_AUTOMAKE(libevent,2.0.21-stable)
-AM_CONFIG_HEADER(config.h)
+AM_INIT_AUTOMAKE
+AC_CONFIG_HEADERS([config.h])
 AC_DEFINE(NUMERIC_VERSION, 0x02001500, [Numeric representation of the version])
 
 dnl Initialize prefix.
--- libevent-2.0.21-stable/test/Makefile.am.bero	2013-01-14 00:36:52.235571546 +0100
+++ libevent-2.0.21-stable/test/Makefile.am	2013-01-14 00:36:59.162486677 +0100
@@ -19,7 +19,7 @@ endif
 EXTRA_PROGRAMS = regress
 noinst_HEADERS = tinytest.h tinytest_macros.h regress.h tinytest_local.h
 
-TESTS = $(top_srcdir)/test/test.sh
+TESTS = test.sh
 
 BUILT_SOURCES =
 if BUILD_REGRESS
